src/SmartComponents/SystemTable/SystemTable.js

```diff
@@ -79,6 +91,40 @@ const SystemTable = ({
       };
     });
 
+  const eligibilityFilter = {
```

Comment:
Nitpick: It would be maybe better if we could extract these filter definitions into a separate file so that they can be shared in the future.

Reply:
I'll leave them in the current file because they are closely coupled to the SystemTable component.  They didn't seem like 'helpers' or 'constants' either.  If they need to be reused elsewhere then I'll move them then.